behave/setup.cfg

```diff
@@ -1,7 +0,0 @@
-[behave.userdata]
```

Comment:
Can we remove this file if it's just going to be empty?

Reply:
@Pheonnexx @psweeting1 I left this in as I thought it might be required by behave, do you know? Can removed if not needed.

Reply:
removed the file